include/motor-control/core/tasks/motor_hardware_task.hpp

```diff
@@ -85,7 +83,7 @@ class PipetteMotorHardwareTask {
     }
 
   private:
-    motor_hardware::PipetteStepperMotorHardwareIface* hardware;
+    PipetteStepperHardware* hardware;
```

Comment:
looking at this file again, if you made all the hardwares required to have a `periodic_tasks()` method that did basically `read_estop_in()`, `read_limit_switch()`, `read_sync_in()`, and in the case of pipette `read_tip_sense()` you could get rid of the pipette motor task

Reply:
Same comment as below, but:

The way that we're doing tip sensing is going to change so the need for a pipette hardware class seems moot to me anyways.

I didn't want to address that in this PR since it's already a large-ish refactor. I'll make a ticket to track the work.